packages/rsbuild-plugin-angular/src/lib/plugin/utils/devkit-fialing-import.unit.test.ts

```diff
@@ -0,0 +1,19 @@
+import { describe, expect } from 'vitest';
+
+vi.mock('@angular/compiler-cli', async () => {
+  const actual = await vi.importActual('@angular/compiler-cli');
+  return {
+    ...actual,
+    VERSION: {
+      major: 14,
+    },
+  };
+});
+
+describe('devkit importing an angular version <15', async () => {
+  it('should throw an error when its import is resolved', async () => {
+    await expect(() => import('./devkit.ts')).rejects.toThrowError(
+      'AnalogJS is not compatible with Angular v14 and lower'
```

Comment:
Should update the error - this isn’t Analog :)

it’s my bad for missing it

Reply:
Updated the error msg